### DISCUSSION TITLE
'MODULE_NOT_FOUND' occur when using multiple config files

### DISCUSSION THREAD

**Discussion by bpgity** [webpack/webpack#16712] [Q&A]
We are using multiple config files for webpack configurations.

1) DEV - webpack.config.dev.js
2) PROD - webpack.config.prod.js

After running project on dev config getting error 'MODULE_NOT_FOUND' 

D:\react\reactwebpackdemo>**npm run serve**

> reactwebpackdemo@1.0.0 serve
> **webpack serve --config webpack.config.dev.js**

[webpack-cli] Failed to load 'D:\react\reactwebpackdemo\webpack.config.dev.js' config
[webpack-cli] Error: Cannot find module 'D:\react\reactwebpackdemo\webpack.config.dev.js'
Require stack:
- D:\react\reactwebpackdemo\node_modules\webpack-cli\lib\webpack-cli.js
- D:\react\reactwebpackdemo\node_modules\webpack-cli\lib\bootstrap.js
- D:\react\reactwebpackdemo\node_modules\webpack-cli\bin\cli.js
- D:\react\reactwebpackdemo\node_modules\webpack\bin\webpack.js
    'D:\\react\\reactwebpackdemo\\node_modules\\webpack-cli\\lib\\bootstrap.js',
    'D:\\react\\reactwebpackdemo\\node_modules\\webpack-cli\\bin\\cli.js',
    'D:\\react\\reactwebpackdemo\\node_modules\\webpack\\bin\\webpack.js'
  ]
}

I am using **windows** and below is my webpack versions

"webpack": "^5.75.0",
"webpack-cli": "^5.0.1",
"webpack-dev-server": "^4.11.1"

**Comment by alexander-akait** [comment]
Do you have `webpack.config.dev.js` file?
